examples/Application/Traffic-Light-Control/examples/config_hz_1.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
can we rename the folder examples as `scenarios` ?

Reply:
yes, fixed.